R/pkg/R/functions.R

```diff
@@ -679,6 +679,19 @@ setMethod("hash",
             column(jc)
           })
 
+#' @details
+#' \code{data_type}: Returns the data type of a given column.
+#'
+#' @rdname column_misc_functions
+#' @aliases data_type data_type,Column-method
+#' @examples \dontrun{data_type(df$c)}
```

Comment:
example should no be added here - see the example for `hash`

Reply:
see this line of the code example of hash
https://github.com/mmolimar/spark/blob/ed52e2f856f78fb2dca23b6be2f682caa0a88c81/R/pkg/R/functions.R#L176